app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -123,6 +127,19 @@ public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
                     stopProgressLoop();
                 }
             }
+
+            @Override
+            public void onPositionDiscontinuity(Player.PositionInfo oldPosition, Player.PositionInfo newPosition, int reason) {
+                // discontinuity for reason internal usually indicates an error, and that the player will reset to its default timestamp
+                if (reason == Player.DISCONTINUITY_REASON_INTERNAL) {
+                    Timber.d("caught player discontinuity (reason internal) - oldPos %s newPos %s", oldPosition.positionMs, newPosition.positionMs);
+                }
+            }
+
+            @Override
+            public void onTimelineChanged(Timeline timeline, int reason) {
+                Timber.d("caught player timeline changed! reason %s", reason == 0 ? "PLAYLIST_CHANGED" : "SOURCE_UPDATE");
```

Comment:
Probably better to compare with TimelineChangeReason here

https://exoplayer.dev/doc/reference/com/google/android/exoplayer2/Player.html#TIMELINE_CHANGE_REASON_PLAYLIST_CHANGED

Reply:
done